include/circt/Dialect/FIRRTL/FIRRTLOpInterfaces.td

```diff
@@ -363,6 +363,16 @@ def FInstanceLike : OpInterface<"FInstanceLike", [
   let description = "Provide common  instance information.";
 
   let methods = [
+    InterfaceMethod<"Get the name of the instantiated module",
```

Comment:
Why aren't these methods in InstanceGraphInstanceOpInterface?  And if they are, they shouldn't be redeclared here, the interface should inherit them.

Reply:
In this particular case, I don't think utility methods should be in the instance graph op interface. Since it bridges multiple dialects, I'd like it to enforce a minimal number of constraints on the ops which implement it.

On a more realistic note, I tried doing that, but the list of errors lit up like a Christmas tree. I'll make a note of this and consider moving this method to the op interface in a subsequent PR.